src/App.tsx

```diff
@@ -18,7 +18,8 @@ function App() {
       {errorMessage && <ErrorBanner />}
       <BrowserRouter>
         <Routes>
-          <Route path="/" element={<Home />} />
+          <Route path="/home/*" element={<Home />} />
```

Comment:
how it's working?

Reply:
documentation said to put an asterisk in parents route path when route definitions are split in multiple files